client/sites/components/guides/sites-guide.tsx

```diff
@@ -0,0 +1,71 @@
+import { Guide } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import guideStep1 from 'calypso/assets/images/sites/sites-guide-1.png';
+import guideStep2 from 'calypso/assets/images/sites/sites-guide-2.png';
+import './sites-guide.scss';
+
+const storageKey = `sites_guide_is_dismissed`;
+
+const SitesGuide = () => {
+	const [ isOpen, setIsOpen ] = useState( true );
+	const [ isImageLoading, setIsImageLoading ] = useState( true );
+
+	if ( ! isOpen || localStorage.getItem( storageKey ) ) {
+		return null;
+	}
+
+	const dismiss = () => {
+		localStorage.setItem( storageKey, 'true' );
+
+		setIsOpen( false );
+	};
+
+	return (
+		<Guide
```

Comment:
Can we use the reusable `AnnouncementModal` component instead of building a new one?

See https://wpcalypso.wordpress.com/devdocs/blocks/announcement-modal

A good thing about that component is that it stores the dismissal in [a user preference](https://github.com/Automattic/wp-calypso/blob/6c4f1c894bdb001a0cb353d814a80181f42649c4/client/blocks/announcement-modal/index.jsx#L42) instead of in the browser local storage, so the modal is dismissed across all browsers.

Reply:
Actually, it seems that the design of that modal deviates from the default Core Guide, so maybe it's better to stick to the current approach.

I'd reconsider though changing the dismissal in a user preference.

Reply:
Updated to store in preferences.